debugger/src/main/resources/acceptable-security-check-failures.txt

```diff
@@ -16,3 +16,9 @@ org\.apache\.pdfbox\.pdmodel\.font\.FileSystemFontProvider.*
 # continue with a valid default for Windows
 java\.io\.FilePermission ".*somaxconn", "read"
 io\.netty\.util\.NetUtil\$.*:267
+
+# At startup time, Karaf tries to create a system terminal and will fall back to a dumb one of it c
```

Comment:
Good question. I don 't really know. I just know all our itests have always fallback to dumb terminal. But in this case, it ain't a security failure since the code is designed to handle this type of security exception.

Reply:
But what is dumb terminal? That's what I'd like to know.

Reply:
? This is what Karaf calls it ...